src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -94,6 +105,33 @@ public List<AudioFileInfo> processAudioForPlaybacks(Map<MediaEntity, Path> media
         return concatenatedAndMergedAudioFileInfos;
     }
 
+    private boolean isWellFormedAudio(List<AudioFileInfo> audioFileInfos) {
```

Comment:
gonna leave as is. there's 2 reasons:
1) if there's only 1 channel for audio data then it can be considered "well formed" regardless of any other criteria
2) if all the counts for start times match - it's only "well formed" if none of the counts = 1